docs/release-notes/cloud/cloud-2024-01-12-gh-repository-ssh-keys-discontinuation.md

```diff
@@ -0,0 +1,83 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+---
+
+# Discontinuation of SSH keys for GitHub repositories January 12, 2024
```

Comment:
@claudiacarpinteiro When I read the preview I though we are saying that this will happen on the January 12! 
Perhaps is the message not communicating what's really happening. Because in the table bellow it's different. 
- I'll leave it to you to decide tho.

Reply:
I'll keep "discontinuation", as talked.